src/session_impl.cpp

```diff
@@ -633,9 +633,7 @@ namespace aux {
 		update_dht();
 		update_peer_fingerprint();
 		update_dht_bootstrap_nodes();
-#ifndef TORRENT_DISABLE_DHT
 		update_dht_announce_interval();
```

Comment:
is this change significant? I wouldn't expect this function to do anything when DHT is disabled